internal/project/golang/generate.go

```diff
@@ -312,5 +318,51 @@ func (generate *Generate) CompileDockerfile(output *dockerfile.Output) error {
 		}
 	}
 
+	if generate.versionPackagePath() != "" {
+		output.Stage("embed-generate").From("tools").
+			Step(step.Arg("SHA")).
+			Step(step.Arg("TAG")).
+			Step(step.Arg("NAME")).
```

Comment:
how is `NAME` being used?

Reply:
Thanks. Leftover from the previous iteration. Removed.